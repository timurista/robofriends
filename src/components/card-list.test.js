// import * as React from 'react';
// import { shallow, render } from 'enzyme'
// import CardList from './card-list'


describe('CardList component', () => {
  const intialProps = {
    robots: [
      { id: 1, name: 'John Snow', username: 'johnjohn', email: 'john@email.com'}
    ]
  }
  const setup = (props: {}) => {
    const newProps = {...intialProps, ...props};
    // return shallow(<CardList {...newProps}/>)
  }

  it('CardList component exists', () => {
    const wrapper = setup();
    // expect(wrapper.debug()).toMatchSnapshot();
  })
})