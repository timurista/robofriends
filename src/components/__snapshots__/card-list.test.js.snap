// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList component CardList component exists 1`] = `
"<div>
  <Card id={1} name=\\"John Snow\\" username=\\"johnjohn\\" email=\\"john@email.com\\" />
</div>"
`;
